gammapy/irf/background.py

```diff
@@ -136,6 +134,63 @@ def to_2d(self):
     def peek(self, figsize=(10, 8)):
         return self.to_2d().peek(figsize)
 
+    def plot_at_energy(self, energy=None, ax=None, add_cbar=True, ncols=3, **kwargs):
+        """ Plot the background rate in Field of view co-ordinates at a given energy.
+
+        Parameters
+        -----------
+        energy : `~astropy.units.Quantity`
+            list of Energy
+        ax: `~matplotlib.axes.Axes`, optional
+            Axis
+        add_cbar : bool
+            Add color bar?
+        ncols : int
+            Number of columns to plot
+        **kwargs : dict
+            Keyword arguments passed to `~matplotlib.pyplot.pcolormesh`.
+        """
+        import matplotlib.pyplot as plt
+
+        n = len(energy)
+        cols = min(ncols, n)
+        rows = 1 + (n - 1) // cols
+        width = 12
+        figsize = (width, width * rows / cols)
```

Comment:
I think it's important to keep `figsize` as parameter to the function, however you can set a default just like in `Map.plot_grid()`

Reply:
Done